unikernel.ml

```diff
@@ -928,32 +910,13 @@ struct
             in
             new_user_cookies ~user ~filter ~redirect store reqd
         | None ->
-            let error =
-              {
-                Utils.Status.code = 404;
-                title = "Not Found";
-                success = false;
-                data = `String "Auth cookie not found";
-              }
-            in
-            reply reqd ~content_type:"text/html"
-              (Guest_layout.guest_layout ~page_title:"404 | Mollymawk"
-                 ~content:(Error_page.error_layout error)
-                 ~icon:"/images/robur.png" ())
-              `Not_found)
+            Middleware.http_response ~api_meth:false reqd ~title:"Error"
+              ~data:(`String "Authentication cookie not found.") `Not_found)
     | Error (`Msg err) ->
-        let error =
-          {
-            Utils.Status.code = 400;
-            title = "Bad Request";
-            success = false;
-            data = `String err;
-          }
-        in
-        reply reqd ~content_type:"text/html"
-          (Guest_layout.guest_layout ~page_title:"400 | Mollymawk"
-             ~content:(Error_page.error_layout error)
-             ~icon:"/images/robur.png" ())
+        Middleware.http_response ~api_meth:false reqd ~title:"Error"
```

Comment:
I'm as well curious whether the Guest_layout.guest_layout is now dropped or not, and whether the content: (Error_page.error_layout error) is now dropped?